### PR TITLE
Ui improved of website

### PR DESCRIPTION
**Pull request by arpit456jain**
## Related Issuse
Ui improved of website

Closes: #91 

#### Describe the changes you've made

https://drive.google.com/file/d/1g5i6xt__uNyma-Hjobs1VmXxNzMybr7i/view?usp=sharing
## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

## Screenshots
screencapture-127-0-0-1-5000-2021-04-25-21_24_41

### CODE REVIEW

**Comment by arpit456jain** [comment]
@kritikaparmar-programmer kindly review my PR

**Review by utkarsh0702 — APPROVED** [review]

File: templates/index.html [hunk 1]
```diff
@@ -76,9 +104,8 @@ <h6 class="card-text"></h6>
         </div>
       </div>
```
Comment (kritikaparmar-programmer) [inline templates/index.html:106]: one closing div tag missing here.

**Review by kritikaparmar-programmer — CHANGES_REQUESTED** [review]
The rest looks good to me.

File: templates/index.html (hunk 1 above)
Comment (arpit456jain) [reply templates/index.html:106]: Ok let me check

**Comment by arpit456jain** [comment]
screencapture-127-0-0-1-5000-2021-04-29-12_16_24

@kritikaparmar-programmer

**Review by kritikaparmar-programmer — APPROVED** [review]

**Comment by arpit456jain** [comment]
@kritikaparmar-programmer kindly add suitable labels to it 😅😅😅😅

**Comment by arpit456jain** [comment]
thanks

**Pull request merged**